portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Olu's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="row">
+      <img src="images/logo.png"
+    </div>   
+    <ul class="main-nav">    
+      <li><a href="index.html" style="color:white"> HOME </a></li>
+      <li><a href="gallery.html" style="color:white"> PHOTOGRAPHY </a> </li>
+	  <li><a href="showcase.html" style="color:white"> PROJECT SHOWCASE </a></li>
```

Comment:
nit: This line has some extra indentation.

Reply:
fixed

Reply:
Seems like the change wasn't included in the most recent commit?